src/Modules/FreeModules-graded.jl

```diff
@@ -300,7 +300,8 @@ mutable struct BiModArray{T}
       r.SF = parent(s[1])
     end
     r.S = s
-    r.O = Array{FreeModuleElem_dec{S}, 1}(undef, Singular.ngens(s))
+    #r.O = Array{FreeModuleElem_dec{S}, 1}(undef, Singular.ngens(s))
+    r.O = map(x -> convert(F,x), [s[i] for i=1:Singular.ngens(s)])
```

Comment:
Having both a `map` and an array comprension increases complexity for the reader, what about 
```suggestion
    r.O = [convert(F, s[i]) for i=1:Singular.ngens(s)]
```
?

Reply:
Yes, that's better. That was a bit stupid from my side.